ApiReview/API.Polly.Core/NoDocs/Polly.Timeout/OnTimeoutArguments.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Polly.Timeout;
+
+public readonly struct OnTimeoutArguments
+{
+    public ResilienceContext Context { get; }
```

Comment:
there is no consistency across various On*Arguments types on whether `ResilienceContext` should be included in the args or not.

Reply:
The `Context` property is on all outcome-less arguments. All outcome-based delegates use `OutcomeArguments` that enforces the `ResilienceContext` and `Outcome` on construction.

Reply:
Let's try to get rid of `OutcomeArguments`